facade/nodeFacade.go

```diff
@@ -321,7 +321,10 @@ func (nf *nodeFacade) GetLastPoolNonceForSender(sender string) (uint64, error) {
 
 // GetTransactionsPoolNonceGapsForSender will return the nonce gaps from pool for sender, if exists, that is to be returned on API calls
 func (nf *nodeFacade) GetTransactionsPoolNonceGapsForSender(sender string) (*common.TransactionsPoolNonceGapsForSenderApiResponse, error) {
-	accountResponse, _, err := nf.node.GetAccount(sender, apiData.AccountQueryOptions{})
+	ctx, cancel := nf.getContextForApiTrieRangeOperations()
```

Comment:
I think here we can remove the getContextForApiTrieRangeOperations as this call does not support options.

Reply:
refactored to have a method GetAccountWithKeys